### PR TITLE
DOC: update description of the Dirichlet distribution

### PR DESCRIPTION
**Pull request by tuelwer**
Clean up description of the Dirichlet distribution. Fixed notation and added some details.

### CODE REVIEW

File: numpy/random/mtrand/mtrand.pyx [hunk 1]
```diff
@@ -4653,14 +4653,15 @@ cdef class RandomState:
 
     def dirichlet(self, object alpha, size=None):
         """
-        dirichlet(alpha, size=None)
+        dirichlet (alpha, size=None)
```
Comment (bashtage) [inline numpy/random/mtrand/mtrand.pyx]: Did you mean to put a space here?  Normally this should match the function declaration omitting self.

Comment (tuelwer) [reply numpy/random/mtrand/mtrand.pyx]: No, I did not do this on purpose. I just wanted to edit the documentation.

**Comment by mattip** [comment]
LGTM. It would be nice to add a reference.

**Comment by mattip** [comment]
Thanks @tuelwer

**Pull request merged**